### PR TITLE
Format improvements in Kernel.SpecialForms.<<>>/2 docs

### PR DESCRIPTION
**Pull request by eksperimental**
Adds backticks and italics

### CODE REVIEW

**Comment by whatyouhide** [comment]
I am on board with the backticks but definitely against the italics as I don't feel they add anything to the documentation. In any case, we do italics with `*`, not `_` (`*foo*` instead of `_foo_`) in case other people like the italics :)

File: lib/elixir/lib/kernel/special_forms.ex [hunk 1]
```diff
@@ -233,13 +233,13 @@ defmodule Kernel.SpecialForms do
...
   representation.
 
-  Modifier             | Relevant Type(s)
-  -------------------- | ----------------
-  `signed`             | `integer`
-  `unsigned` (default) | `integer`
-  `little`             | `integer`, `float`, `utf16`, `utf32`
-  `big` (default)      | `integer`, `float`, `utf16`, `utf32`
-  `native`             | `integer`, `utf16`, `utf32`
+  Modifier               | Relevant Type(s)
```
Comment (eksperimental) [inline lib/elixir/lib/kernel/special_forms.ex]: the changes in this table only add italics to default, the rest of the changes is to line everything up in the table.

**Comment by josevalim** [comment]
We definitely should not use _ for italics. :)



-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by eksperimental** [comment]
replaced `_` with `*`

**Comment by whatyouhide** [comment]
I still am very much 👎 on using italics in the first place, so that's the reason why I'm not approving the PR :)

**Comment by lexmag** [comment]
Looking at generated documentation with italics, I think it looks just fine without addition of italic.

**Comment by eksperimental** [comment]
I removed italics form "(alias for ...)" since I'm using backticks inside parenthesis, but still left italics for "(default)".

**Comment by whatyouhide** [comment]
I still won't approve because of italics in "(default)" as well, I don't see the point. Do we use it anywhere else? @josevalim, are you explicitly in favor of italics or more indifferent? :)

**Comment by josevalim** [comment]
Let's remove italics given the left side is already in code notation. It would be OK otherwise.

**Comment by eksperimental** [comment]
done

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**